yetanotherco/aligned_layer
# chore: bump up sdk version in zkquiz

# Description

- This PR bumps up the Aligned SDK version to v0.3.0 in zkquiz.

# To Test

- Follow the [instructions](https://github.com/yetanotherco/aligned_layer/blob/main/examples/zkquiz/README.md) to use zkquiz and make sure everything works correctly.
